lib/high_voltage/page_collector.rb

```diff
@@ -0,0 +1,43 @@
+module HighVoltage
+  require 'find'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.